Amazon.IonObjectMapper.PerformanceTest/PerformanceTest.cs

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.IonObjectMapper.PerformanceTest
+{
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.IO;
+
+    [TestClass]
+    public class PerformanceTest
+    {
+        /// <summary>
+        /// Size of dictionary to use as the baseline for determining how much runtime and 
+        /// memory is required for one entry.
+        /// 
+        /// Setting this to be too low will cause overhead runtime and memory to dominate 
+        /// the per-entry baseline.
+        /// </summary>
+        private const long BaseCount = 100000;
+
+        /// <summary>
+        /// The orders of magnitude to test over the BaseCount amount. 
+        /// 
+        /// For example, a BaseCount of 100,000 with a Magnitude of 2 will test 
+        /// 1,000,000 and 10,000,000 entries.
```

Comment:
As of now, the perf test is testing 100,000, 1,000,000, and 10,000,000 entries. 
Please update this comment or update the loop to match this comment.

Reply:
It's technically accurate as is since 100000 isnt being tested, but being used as the baseline.

Regardless, I updated it.